build.gradle

```diff
@@ -40,14 +40,6 @@ subprojects {
 
     [compileJava, compileTestJava, javadoc]*.options*.encoding = 'UTF-8'
 
-    // Set up checkstyle
-    apply plugin: 'checkstyle'
-    def checkstyleConfigDir = new File(rootDir, "config/checkstyle")
-    checkstyle {
-        configFile = new File(checkstyleConfigDir, "checkstyle.xml")
-        configProperties.checkstyleConfigDir = checkstyleConfigDir
-    }
-
```

Comment:
is this preventing the build? If not let's not do this in this PR, we want to enable checkstyle, not disable it.